tortoise/queryset.py

```diff
@@ -426,6 +426,33 @@ def exclude(self, *args: Q, **kwargs: Any) -> "QuerySet[MODEL]":
         """
         return self._filter_or_exclude(negate=True, *args, **kwargs)
 
+    def _parse_orderings(
+        self, orderings: Tuple[str, ...], reverse=False
+    ) -> List[Tuple[str, Order]]:
+        """
+        Convert ordering from strings to standard items for queryset.
+
+        :param orderings: What columns/order to order by
+        :param reverse:  Whether reverse order
+        :return: standard ordering for QuerySet.
+
+        :raises FieldError: If a field provided does not exist in model.
+        """
+        if not orderings:
+            raise FieldError("No fields passed")
```

Comment:
I think it would be better to change `latest` and `earliest` signatures to avoid raising this exception at all.

```python
def earliest(self, ordering: str, *orderings: str) -> QuerySetSingle[Optional[MODEL]]:
  queryset = self._clone()
  queryset._orderings = self._parse_orderings(ordering + orderings, reverse=True)
  return queryset._as_single()
```

This way the signature itself will make at least one order argument required. It will help to avoid having this check in  `order_by`: 

`queryset._orderings = self._parse_orderings(orderings) if orderings else []`  

because `_parse_orderings` won't be raising exceptions anymore.

Reply:
`ordering: str, *orderings: str` style is not readability, so I move `not orderings` check back to latest/earliest and remove the if-else in `order_by`

Reply:
Alright, works for me!